src/store/Store.js

```diff
@@ -45,6 +46,11 @@ export default class Store {
 			this.setState({ openSessionIds: [sessionId, ...openSessionIds] });
 		});
 
+		window.addEventListener('visibilitychange', () => {
+			!this._state.minimized && this._state.visible && parentCall('openWidget');
+			this._state.iframe.visible ? parentCall('showWidget') : parentCall('hideWidget');
```

Comment:
Why are you checking `this._state.iframe.visible` here and checked `this._state.visible` above?

Reply:
`this._state.visible` isn't been changed, it's unnecessary this check.